y_pef_schedule.install

```diff
@@ -27,16 +27,16 @@ function y_pef_schedule_uninstall(): void {
 /**
  * Creates required fields for the module.
  *
- * This function defines and saves the 'field_session_color' field storage and
- * field config for nodes of type 'session', which is used to store color
- * information for sessions.
+ * This function defines and saves the 'field_activity_color' field storage and
+ * field config for nodes of type 'activity', which is used to store color
+ * information for activitys.
  *
  * @throws \Drupal\Core\Entity\EntityStorageException
  */
 function _y_pef_schedule_create_fields(): void {
-  // Define and save field storage for 'Session color'.
+  // Define and save field storage for 'activity color'.
   FieldStorageConfig::create([
-    'field_name' => 'field_session_color',
+    'field_name' => 'field_activity_color',
     'entity_type' => 'node',
     'type' => 'string',
     'settings' => [
```

Comment:
Let's reuse colorapi for better CX, please

Reply:
Done